packages/next/package.json

```diff
@@ -39,12 +39,14 @@
     "chalk": "^4.1.0",
     "copy-webpack-plugin": "^10.2.4",
     "dotenv": "~10.0.0",
-    "express": "^4.18.1",
```

Comment:
Good to remove since we don't need this anymore. 
`custom-server` generator also imports `next` instead of `express` so this should be good.